bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2260,8 +2264,82 @@ namespace {
       return BindCppObject( addr, (Cppyy::TCppType_t)Cppyy::GetScope( "TObject" ), kFALSE );
    }
 
+   //- Adding array interface to classes ---------------
+   void AddArrayInterface(PyObject *pyclass, PyCFunction func)
+   {
+      Utility::AddToClass(pyclass, "_get__array_interface__", func, METH_NOARGS);
+      Utility::AddProperty(pyclass, "_get__array_interface__", "__array_interface__");
+   }
+
+   PyObject *FillArrayInterfaceDict(UInt_t bytes, char type)
+   {
+      PyObject *dict = PyDict_New();
+      PyDict_SetItemString(dict, "version", PyLong_FromLong(3));
+#ifdef R__BYTESWAP
+      const char endianess = '<';
+#else
+      const char endianess = '>';
+#endif
+      PyDict_SetItemString(dict, "typestr",
+                           PyROOT_PyUnicode_FromString(TString::Format("%c%c%i", endianess, type, bytes).Data()));
```

Comment:
Here is the change. `PyString_FromString` translates to python3 as `PyUnicode_FromString`, which is wrapped by `PyROOT_PyUnicode_FromString` with precompiler statements in `PyROOT.h`.

Reply:
Hi @stwunsch thanks for the changes! Are we then good to go with this?

Reply:
Not yet, as written above `ROOT.std.vector("float")(1)` fails to run on python3 and I've not yet evaluated whether this is due to the changes in this PR.